network-api/networkapi/wagtailpages/templates/wagtailpages/blog_page.html

```diff
@@ -2,6 +2,22 @@
 
 {% load wagtailcore_tags mini_site_tags %}
 
+{% if coralTalkServerUrl %}
+  {% block coral_talk_meta %}
+    <meta property="article:author" content="{{ page.author }}" />
+    <meta property="article:published_time" content="{{ page.first_published_at|date:"F j, Y" }}">
+    {% if page.last_published_at %}
+      <meta property="article:modified" content="{{ page.last_published_at|date:"F j, Y" }}"/>
+    {% endif %}
+    {% with category=page.specific.category.first %}
+      {% if category %}
+        <meta property="article:section" content="Category - {{ category }}" />
```

Comment:
I'm not sure we need to prefix the content with "Category  - ", unless there's a reason that I'm missing

Reply:
I didn't think article section was representative of the actual content value but, I can definitely remove that, np.